pl/reference/tutorial.rst

```diff
@@ -72,7 +72,7 @@ Zarządzanie zależnościami
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 Bardzo ważnym pojęciem, które musi być zrozumiane podczas pracy z Phalconem jest jego :doc:`dependency injection container <di>`. Może to brzmieć bardzo skomplikowanie, ale jest bardzo proste i praktyczne.
 
-A service container is a bag where we globally store the services that our application will use to function. Each time the framework requires a component, it will ask the container using an agreed upon name for the service. Since Phalcon is a highly decoupled framework, :doc:`Phalcon\\Di <../api/Phalcon_Di>` acts as glue facilitating the integration of the different components achieving their work together in a transparent manner.
+Kontener Zależności jest zbiorem, który globalnie przechowuje wszelkie serwisy, z których nasza aplikacja korzysta w danych funkcjach. Za każdym razem gdy Framework wymaga któregoś komponentu, zapyta Kontener o uzgodnioną nazwę danego serwisu. Od kiedy Phalcon jest wysoce rozczłonkowanym Frameworkiem, Kontener Zależności pełni rolę kleju, który łączy różne komponenty osiągając ich synergię.  
```

Comment:
Thanks Jurigag for your personal opinion.
Instead of word 'rozczłonkowanym' I can write 'rozłączonym', according to this [dictionary](http://pl.bab.la/slownik/angielski-polski/decoupled). I like the 'Science language', I like also reading a lot of different kinds of books which have a big influence for my life. I like also writing some novels etc. in my free time. 
I also think that even some a bit 'science' translate of Phalcon's doc in polish language is very good for polish Programmers who can't read english documentation.